Dockerfile

```diff
@@ -1,3 +1,6 @@
+ARG BUILDPLATFORM
```

Comment:
copy past test code?

Reply:
No. Some of it from the docket docs

Reply:
https://docs.docker.com/build/building/multi-platform/